src/core.ts

```diff
@@ -182,7 +182,7 @@ export const container: <M>(def: ContainerDef<M>) => Container<M> = withEnvironm
  */
 export const isolate = <M>(ctr: Container<M>, opts: any = {}): IsolatedContainer<M> => {
   const stateManager = opts.stateManager && always(opts.stateManager) || (() => new StateManager());
-  const env = create({ dispatcher: nthArg(2), effects: new Map(), log: () => {}, stateManager });
+  const env = create({ dispatcher: nthArg(2), effects: new Map(), log, stateManager });
```

Comment:
I'm fine with making this configurable through `opts`, but changing the default behavior is just going to create a bunch of console noise during testing.